.pre-commit-config.yaml

```diff
@@ -28,6 +28,13 @@ repos:
         # Don't commit to main branch.
     -   id: no-commit-to-branch
 
+-   repo: https://github.com/codespell-project/codespell
+    rev: "v2.2.2"
+    hooks:
+    -   id: codespell
+        types_or: [asciidoc, python, markdown, rst]
```

Comment:
Determining the `types` can be de-risked by using the [identity](https://pre-commit.com/#filtering-files-with-types) CLI tool.